### ISSUE TITLE
Make action page keyboard accessible

### ISSUE THREAD

**Issue by alexelash** [azavea/temperate#902]
With the changes in #886, we lost keyboard accessibility for this page. This should be reintroduced, probably manually since flexbox doesn't play nicely with text-overflow inside buttons.
